relentless/_math.py

```diff
@@ -1,35 +1,51 @@
+"""
+Math functions
+==============
+
+This module contains a interpolator using Akima 1D splines. The function's value
+and derivatives can be evaluated.
```

Comment:
I think go a little more generic. We might reorg `_math` and `_collections`, so something like this would be fine:
```suggestion
This module implements some convenience objects for mathematical operations.
```